### ISSUE TITLE
Accessibility: In Inspect: MSAA, the child items of some controls cannot be displayed, and Children property gets error 

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#4172]
* .NET Core Version: 
.Net 6.0.100-alpha.1.20516.5

* Have you experienced this same bug with .NET Framework?: 
No

* More Info
1. This issue cannot repro on .Net Core 3.1.
2. Affected controls: CheckedListBox, ListBox, ToolStrip, StatusStrip, ContextMenuStrip, BindingNavigator, DataGridView, ListView

**Minimal repro:**
1. Create a Winforms .Net Core app with controls: CheckedListBox/ListBox/ToolStrip/StatusStrip, ContextMenuStrip/BindingNavigator/DataGridView/ListView
2. Added items for these controls
3. Build and run app
4. Open the Inspect tool and observe these controls hierarchy on MSAA mode

**Actual behavior:**
Controls' items cannot be displayed in Inspect: MSAA 
![image](https://user-images.githubusercontent.com/26474449/97540226-4b42f900-19fe-11eb-9133-57420824a1b0.png)

**Expected behavior:**
Controls' items can be displayed in Inspect: MSAA hierarchy 
![image](https://user-images.githubusercontent.com/26474449/97540236-4f6f1680-19fe-11eb-8374-dcee4531ca82.png)

**Comment by merriemcgaw** [comment]
@Olina-Zhang what happens with the UIA option selected? How about Accessibility Insights for Windows? Those should be our priority now, rather than MSAA (which we used to use quite a bit).

**Comment by Olina-Zhang** [comment]
@merriemcgaw  I see. Inspect:UIA, Accessibility Insights are worked well for controls hierarchy. They are our top priority. And Inspect:MSAA testing was from another issue: [4100](https://github.com/dotnet/winforms/issues/4100), and @Tanya-Solyanik @dreddy-work suggest us check all controls to ensure no issue in MSAA. so filed several issues related MSAA.

**Comment by vladimir-krestov** [comment]
It can be fixed by #4168

**Comment by Marie-Shi** [comment]
Verified this issue with .Net 6.0.100-alpha.1.20562.2 from master branch, it is **fixed**, now controls' items can be displayed in Inspect: MSAA hierarchy.
![list](https://user-images.githubusercontent.com/45864017/99021301-5c157200-259b-11eb-8f37-13a7daaab4d4.png)

**Comment by Zheng-Li01** [comment]
Verified this issue in .Net SDK 5.0.101 build, this issue is fixed.
